ui/sortable.js

```diff
@@ -105,19 +106,16 @@ return $.widget("ui.sortable", $.ui.mouse, {
 	},
 
 	_setHandleClassName: function() {
-		this.element.find( ".ui-sortable-handle" ).removeClass( "ui-sortable-handle" );
+		var that = this;
+		this._removeClass( this.element.find( ".ui-sortable-handle" ), "ui-sortable-handle" );
 		$.each( this.items, function() {
-			( this.instance.options.handle ?
-				this.item.find( this.instance.options.handle ) : this.item )
-				.addClass( "ui-sortable-handle" );
+			that._addClass( this.instance.options.handle ?
+				this.item.find( this.instance.options.handle ) : this.item,
```

Comment:
I'd actually change this to:

```
that._addClass(
    this.instance.options.handle ?
        this.item.find( this.instance.options.handle ) :
        this.item,
    "ui-sortable-handle"
);
```

It's much easier to see where the parameters change.